src/index.css

```diff
@@ -1,6 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```

Comment:
This won't work that way, i don't believe. You'll need to add that as a script tag to the index.html file in the public folder, inside of the <head> meta tags, for the short and simple way. The other way is to bring it in and reference it locally, which is "better" but totally not a requirement for this right now.